quic/s2n-quic-transport/src/space/mod.rs

```diff
@@ -155,6 +166,8 @@ impl<Config: endpoint::Config> PacketSpaceManager<Config> {
             application: None,
             zero_rtt_crypto: None,
             handshake_status: HandshakeStatus::default(),
+            server_name: None,
+            application_protocol: Bytes::from_static(&[]),
```

Comment:
is there a difference between this and `Bytes::new()` (which doesnt allocate) and why is this preferable (i took this form conn impl)?

Reply:
I think you're right; better to use `Bytes::new()`